### DISCUSSION TITLE
유니티로 빌드잇 로더 사용 시 오브젝트 위치이동 문제가 발생합니다

### DISCUSSION THREAD

**Discussion by oqq59** [naverz/zepeto-studio-kor#237] [5. Q&A]
안녕하세요. 유니티로 빌드잇 로더 사용 시 문제가 있어 질문 올립니다.
기존에 빌드잇으로 제작했던 맵을 유니티로 불러오는건 되는데 빌드잇에서 만든 상태 그대로가 아닌 변형된 상태로 로딩됩니다. 오브젝트 다수의 위치가 랜덤하게 이탈하여 다른 위치에 불러와지는데 해결방법이 있을까요? 깃허브에 같은 문제로 질문한 내용이 없어 직접 작성합니다.

**Comment by moondory77-naverz** [comment]
안녕하세요.
우선 빌드잇 로더를 통해 제작한 맵을 로드했을 때, 
오브젝트의 Transfom이 변형된 상태로 로드되는 현상은 정상적인 상황은 아닌듯 합니다.

상황을 좁히기 위해 몇가지 확인을 여쭙겠습니다.

1. 특정맵을 로드할 때만 발생하는 상황이실까요? 
2. transform의 위치가 변하는 오브젝트 목록에 일관성이 있을까요? (특정 오브젝트에만 발생하는 지 )

**Reply by oqq59** [reply]
답변 감사합니다.

1. 특정 맵에서만 발생하지 않고 두 개의 계정에서 빌드잇을 불러왔을 때 두 계정 모두 동일한 문제가 발생했습니다.
2. 지금까지 확인해본 바 일정하지 않고 랜덤하게 발생하는 듯 합니다. 

빌드잇으로 제작했던 맵을 월드로 확장할 계획이 있어 해결이 간절한 상홥입니다...

**Reply by moondory77-naverz** [reply]
넵 이슈 확인하고 답변드리겠습니다.

**Reply by oqq59** [reply]
감사합니다!
